worker-testing-integration/pom.xml

```diff
@@ -35,22 +35,10 @@
         <dependency>
             <groupId>com.github.workerframework</groupId>
             <artifactId>worker-api</artifactId>
-            <version>3.2.0-SNAPSHOT</version>
-        </dependency>
-        <dependency>
-            <groupId>com.github.workerframework</groupId>
-            <artifactId>util-rabbitmq</artifactId>
-            <version>3.2.0-SNAPSHOT</version>
-        </dependency>
-        <dependency>
-            <groupId>com.github.workerframework</groupId>
-            <artifactId>worker-core</artifactId>
-            <version>3.2.0-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>com.github.workerframework</groupId>
             <artifactId>worker-testing-util</artifactId>
-            <version>3.2.0-SNAPSHOT</version>
```

Comment:
Should this have scope of test?

Reply:
As per build log
http://sou-jenkins2.hpeswlab.net/job/WorkerFramework/job/WorkerFramework~worker-framework~develop~CI/269/consoleFull
It is not reported to have scope "test". Wherever worker-testing-integration is referred,  worker-testing-integration is declared as "test" scope. 
I tried to change to "test" scope and it is throwing compilation errors in worker-example-container/ExampleTestControllerProvider class. So it should not be "test" scope.